thorlcr/slave/slavmain.cpp

```diff
@@ -511,6 +511,25 @@ class CJobListener : public CSimpleInterface
                         }
                         break;
                     }
+                    case DebugRequest:
+                    {
+                        StringAttr jobKey;
+                        msg.read(jobKey);
+                        PROGLOG("DebugRequest: %s", jobKey.get());
```

Comment:
minor: I'd combine this line of tracing together with the one in the if (job) block below.